src/js/extendscript.tsx

```diff
@@ -1,34 +1,14 @@
-import {
-  inCEPEnvironment,
-  loadExtendscript,
-  getExtensionPath,
-  getHostEnvironment,
-  RGBColor
-} from "cep-interface";
-
-if (inCEPEnvironment()) {
-  // @ts-ignore
-  const os = window.cep_node.require("os");
-  // @ts-ignore
-  const fs = window.cep_node.require("fs-extra");
-  // @ts-ignore
-  const path = window.cep_node.require("path");
-  // @ts-ignore
-  const { logger } = require("./logger");
-
-  const extensionPath = getExtensionPath();
-
-  logger.info("start", extensionPath);
-
-  const manifest = fs.readJsonSync(path.join(extensionPath, "manifest.json"));
-  loadExtendscript(manifest["index.jsx.ts"]);
-
-  const host = getHostEnvironment();
-  if (host) {
-    const skin = host.appSkinInfo;
-    const bgColor = skin.panelBackgroundColor.color as RGBColor;
-    document.body.style.background = `rgb(${parseInt(bgColor.red)}, ${parseInt(bgColor.green)}, ${
-      parseInt(bgColor.blue)
-    })`;
-  }
+import { evalExtendscript, getHostEnvironment, RGBColor } from 'cep-interface'
+import { readFileSync } from 'fs'
+
+// @ts-ignore
+const Buffer = cep_node.Buffer
+const jsxBundle = readFileSync('./dist/index.js').toString()
```

Comment:
This breaks the `build` task for me. The `./dist/index.js` file doesn't exist unless I run the `start` task before hand. This was part of the reason why I create the `manifest.json` file to confidently grab the transpiled/hashed parcel bundle names and call them at runtime. Like so:

````js
const manifest = fs.readJsonSync(path.join(extensionPath, "manifest.json"));
loadExtendscript(manifest["index.jsx.ts"]);
````

 I'm a little hesitant to remove this feature.

Reply:
Since we are bundling the ExtendScript file seperately (as the entry file) it will not get a strange suffix, so we can expect it to be named "index.js" in the dist folder.

Also, to get live reloading capabilities, we have to know the path on forehand, since `fs.readJsonSync('some/path')` will be transformed by parcel (at compile time) into a `Buffer(withTheContentsOfTheFile)`, it also works with path.join() but only with variables it can figure out at compile time, so not the manifest file (as it's not there yet)

Reply:
Also, if you want the manifest for other purposes, you can use this plugin: https://www.npmjs.com/package/parcel-plugin-bundle-manifest

Reply:
I'm cool with swapping my built-in manifest implementation out for `parcel-plugin-bundle-manifest` and also maybe removing it from the starter template. But before we decide that, we need to address the error that happens when running `build`. This is the error I get:

````
extendscript.tsx:7:34: ENOENT: nosuch file or directory, open '~/Workspace/parcel-plugin-cep-starter/dist/index.js'
   5 | // @ts-ignore
   6 | var Buffer = cep_node.Buffer;
>  7 | var jsxBundle = fs_1.readFileSync('./dist/index.js').toString();
     |                                  ^
   8 | cep_interface_1.evalExtendscript(jsxBundle);
   9 | var host = cep_interface_1.getHostEnvironment();
````

Are you not seeing the same?

Reply:
I am not seeing the same, but I just pushed a change that will first run the "clean" and "build-jsx" tasks before starting the watch processes (start-jsx & start-js).

Hope that makes it work for you =)

Reply:
Something doesn't feel right with requiring something in `dist` from `src`. I think we want to keep the manifest way of doing this.

Reply:
Unfortunately we cannot use the manifest file I think since we depend on the bundler to insert the contents of the file at compile time, it works like `brfs` (from browserify) if you are familiar with that: https://github.com/browserify/brfs

Without this we cannot have live reload for .jsx files, which is worth the hack I think.

I am not aware of another way of doing this.

Reply:
What if we used https://github.com/kentcdodds/babel-plugin-preval to find the name of the file at compile time?

Reply:
I think adding preval is also a hack, adds another dependency and might be more confusing in the end. My idea about using a process.env variable to get the path is also not going to work since we run 2 different bundlers, 1 for the JS and one for the JSX code, the JS bundler needs to find out the path to the JSX file but that bundler is not aware of the other one, I think we should keep it as it is and add a warning in the readme that when people rename / move the entry file of the extendscript code they also have to update the path in the JS code.